pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -24,6 +24,7 @@ import (
 	ads "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
 	"golang.org/x/time/rate"
 	"google.golang.org/grpc"
+	"k8s.io/apimachinery/pkg/util/uuid"
```

Comment:
Lets not drag any K8S dependency into the proxy/ or networking code. We try to keep all K8S dependencies in the kube packages, because we want others to be able to use these packages elsewhere.